tests/cephfs/cephfs_tier1_ops.py

```diff
@@ -0,0 +1,408 @@
+import logging
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
```

Comment:
Would it be possible to have a meeting on this method? It is a large method which IMHO could be broken into smaller pieces while maintaining the follow. 

In the long run, we would be able to add additional steps/stages without breaking the existing code...